attributes/default.rb

```diff
@@ -45,7 +45,7 @@
 default['cloud']['cloudwatch']['download_url']['centos']   = "https://s3.amazonaws.com/amazoncloudwatch-agent/centos/amd64/#{node['cloud']['cloudwatch']['agent_version']}/amazon-cloudwatch-agent.rpm"
 
 default['cloud']['init']['expat_dir']                      = "#{node['cloud']['init']['install_dir']}/expat"
-default['cloud']['ndb-agent']['version']                   = "0.12.0-SNAPSHOT"
+default['cloud']['ndb-agent']['version']                   = "0.12.0"
```

Comment:
This should be `0.13.0-SNAPSHOT` no?

Reply:
Yeah, but i thought since we haven't added any new changes for 0.11 and 0.12 then i would keep this as 0.12.0 for now and if we plan to merge any fixes we will increment this anyway then

Reply:
Alright